docs-v2/integrations/all/emarsys.mdx

```diff
@@ -89,7 +89,7 @@ _No setup guide yet._
 
 - Emarsys offers different APIs(Emarsys core, Loyalty V4 and SMS), currently Nango support Emarsys core API which offers WSSE over SSL authentication.
 - When creating a new connection in Nango, use a combination of the `username` and `secret` that you received during onboarding. The `username` follows the format `account_name00X`, where X is a digit.
-
+- The Oauth Token has a fetch rate limiting of per `client_id`: 2 requests per minute, 10 requests per hour or per `customer_id`: 10,000 requests per day. For more details [OAuth Token Fetch Rate Limiting](https://dev.emarsys.com/docs/emarsys-core-api-guides/branches/OAuth-2.0/b3c3a1eba8515-authentication-in-v3-api#oauth-token-fetch-rate-limiting)
```

Comment:
```suggestion
- The OAuth Token has a fetch rate limiting of per `client_id`: 2 requests per minute, 10 requests per hour or per `customer_id`: 10,000 requests per day. For more details [OAuth Token Fetch Rate Limiting](https://dev.emarsys.com/docs/emarsys-core-api-guides/branches/OAuth-2.0/b3c3a1eba8515-authentication-in-v3-api#oauth-token-fetch-rate-limiting)
```

Reply:
I have updated this with the latest [commit](https://github.com/NangoHQ/nango/pull/3193/commits/09a7be168ddc42872ab38a7b7dd5a02f4d6b700f)